--- node_modules/ethereum-waffle/dist/esm/deployContract.js     2021-07-08 19:36:00.852652812 +0300
+++ node_modules/ethereum-waffle/dist/esm/deployContract.js     2021-07-08 19:36:31.708671786 +0300
@@ -1,7 +1,7 @@
 import { ContractFactory } from 'ethers';
 import { isStandard, hasByteCode } from './ContractJSON';
 const defaultDeployOptions = {
-    gasLimit: 4000000,
+    gasLimit: 1000000000,
     gasPrice: 9000000000
 };
 export async function deployContract(wallet, contractJSON, args = [], overrideOptions = {}) {
--- node_modules/ethereum-waffle/dist/cjs/deployContract.js     2021-07-08 19:36:00.824645536 +0300
+++ node_modules/ethereum-waffle/dist/cjs/deployContract.js     2021-07-08 19:36:16.128729221 +0300
@@ -3,7 +3,7 @@
 const ethers_1 = require("ethers");
 const ContractJSON_1 = require("./ContractJSON");
 const defaultDeployOptions = {
-    gasLimit: 4000000,
+    gasLimit: 1000000000,
     gasPrice: 9000000000
 };
 async function deployContract(wallet, contractJSON, args = [], overrideOptions = {}) {
